### PR TITLE
postgres 12, 13, 14 packages

### PR DESCRIPTION
**Pull request by rawlingsj**
Signed-off-by: James Rawlings <jrawlings@chainguard.dev>

### CODE REVIEW

File: postgresql-12.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        - "*"
+      attestation: TODO
+      license: BSD
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
```
Comment (imjasonh) [inline postgresql-12.yaml:19]:
```suggestion
```

File: postgresql-13.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        - "*"
+      attestation: TODO
+      license: BSD
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
```
Comment (imjasonh) [inline postgresql-13.yaml:19]:
```suggestion
```

File: postgresql-14.yaml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        - "*"
+      attestation: TODO
+      license: BSD
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
```
Comment (imjasonh) [inline postgresql-14.yaml:19]:
```suggestion
```

File: postgresql-12/postgresql-entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env bash
```
Comment (imjasonh) [inline postgresql-12/postgresql-entrypoint.sh:1]: Hard to tell at a glance how different these files are, and if there might be some way to share them. Is there any benefit to having one `postgresql-entrypoint.sh` that gets referenced by each pipeline?

**Review by kaniini — DISMISSED** [review]

File: postgresql-12/postgresql-entrypoint.sh (hunk 4 above)
Comment (rawlingsj) [reply postgresql-12/postgresql-entrypoint.sh:1]: Yeah they're exactly the same - I did wonder whether to reference a single one or duplicated them for each version as the melange source directory mounts a folder for sources, that matches the package name.  In this postgres case it means a folder name with the version number in it.  

Having one is better though, will switch and have a hard link to the `postgresql-entrypoint.sh`

**Comment by rawlingsj** [comment]
@kaniini  @imjasonh latest commit uses the same entrypoint file for multiple postgres packages.  I did a local test and the default source-dir remains the package name so not to break existing behaviour.

**Review by kaniini — APPROVED** [review]

**Comment by kaniini** [comment]
Is this ready to go in?

**Comment by rawlingsj** [comment]
Yes ready to go.

**Pull request merged**